docs/blockchain-compute/usage/via-command-line/preparations.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Preparations
+id: preparations
+---
+
+# Preparations
+
+## Requirements
+
+To be a consumer, you only need access to the Cudos blockchain, an SSH key, and some CUDOS tokens.
+
+:::info
+If you simply want to purchase a VM, and would prefer to use a UI, you can use the [website](https://compute.cudos.org).
+
+This documentation is only needed if you want to create your own integration with the contract, or just want to learn how the system works.
+:::
+
```

Comment:
should we also mention that the testnet contract can only be used if they are whitelited by a Cudos admin?

Reply:
actually, let's remove the testnet as we'll likely never whitelist anyone external